lib/plug/parsers.ex

```diff
@@ -73,6 +73,9 @@ defmodule Plug.Parsers do
   the map of params parsed by one of the parsers listed in `:parsers` and
   `:params` set to the result of merging the `:body_params` and `:query_params`.
 
+  It is important that `Plug.Parsers` is ordered prior to the `:dispatch` plug
+  otherwise the matched function will not receive the parsed `Plug.Conn`.
```

Comment:
Okay, that makes sense. I'll go ahead and make a doc update to `Plug.Router` that includes this use-case.